test/cpp_extensions/libtorch_agnostic_extension/libtorch_agnostic/ops.py

```diff
@@ -164,3 +164,15 @@ def fill_infinity(t) -> Tensor:
     Returns: The modified tensor (same as input)
     """
     return torch.ops.libtorch_agnostic.fill_infinity.default(t)
+
+
+def test_default_constructor(undefined) -> bool:
```

Comment:
A bit confusing that this takes in "undefined" but the kernel.cpp version takes in defined. We should converge on defined?